frontend/src/layout/Navigation.tsx

```diff
@@ -54,7 +54,17 @@ export const Nav: React.FC<Props> = ({ fragRef }) => {
 
     return <nav>
         {realm.parent !== null && <>
-            <LinkWithIcon to={realm.parent.path} iconPos="left" css={{ padding: "6px 4px" }}>
+            <LinkWithIcon
```

Comment:
Focusing the first item looks like this:

![image](https://user-images.githubusercontent.com/123272/156733461-c20bdedf-8d00-4d26-8d9c-1393102a8682.png)

Is this intended? :thinking: 

Something analogous happens on the last link in the list.

Reply:
Looks nice now! :D